### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by Nkaleth**
Learning objectives of this PR:

- Learning objectives
- Understand the concept of state management.
- Use store, actions, and reducers in React.
- Connect React and Redux.

### CODE REVIEW

File: src/components/ButtonAddnewBook.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const ButtonAddnewBooks = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const title = event.target.title.value;
+    const author = event.target.author.value;
+    const newbook = { title, author };
+    dispatch(AddBook(newbook));
+    event.target.reset();
```
Comment (Whoistolu) [inline src/components/ButtonAddnewBook.jsx]: - [ ] [REQUIRED] Kindly use the useState hook for `title` and `author`, which is updated through the `onChange` event as stated by the project requirement. Such that the initial and final state looks like this :
```jsx
 const [title, setTitle] = useState("")
 const [author, setAuthor] = useState("")
```

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Nkaleth 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] In the `src/components/ButtonAddnewBook.jsx` component, kindly add an `onChange` event to the two input fields to update your state.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Nkaleth  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/ButtonAddnewBook.jsx (hunk 1 above)
Comment (Nkaleth) [reply src/components/ButtonAddnewBook.jsx]: Thank you for taking the time to explain me!

Comment (Whoistolu) [reply src/components/ButtonAddnewBook.jsx]: You are welcome @Nkaleth 👍🏽 
Happy coding 🥇

**Pull request merged**